oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -338,7 +338,7 @@ receiver-meter:
 receiver-otel:
   selector: ${SW_OTEL_RECEIVER:default}
   default:
-    enabledHandlers: ${SW_OTEL_RECEIVER_ENABLED_HANDLERS:"oc"}
+    enabledHandlers: ${SW_OTEL_RECEIVER_ENABLED_HANDLERS:"oc,otlp"}
```

Comment:
We cannot merge `enabledOcRules` and `enabledOtlpRules` because the rules are read by handler name `oc`/`otlp` like `enabled<handlerName>Rules`

Reply:
I think we could make two handlers read the same things?
Do you have concerns?

Reply:
It's hardly possible to do that on the basis of the current code. Read this 

https://github.com/apache/skywalking/blob/dc7c42cd97c37ee377ea144364c15d41c1099864/oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/OtelMetricReceiverConfig.java#L44-L61

The rules are read by handler `getEnabledRulesFrom(String handler)`, so if we want to share the rules for `oc` and `otlp`, we need either:

- hard code the returned rules for `oc` and `otlp` handlers.
- OR, let handler read the rule file on their own, instead of here:

https://github.com/apache/skywalking/blob/dc7c42cd97c37ee377ea144364c15d41c1099864/oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/OtelMetricReceiverProvider.java#L70-L72

Let me try the second way

Reply:
If it is too complex, it is fine to keep for now. Just seem a little strange.